playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -132,6 +132,7 @@ EDXAPP_CAS_ATTRIBUTE_PACKAGE: ""
 EDXAPP_ENABLE_AUTO_AUTH: false
 # Settings for enabling and configuring third party authorization
 EDXAPP_ENABLE_THIRD_PARTY_AUTH: false
+EDXAPP_ENABLE_OAUTH2_PROVIDER: false
```

Comment:
Why do your changes here (adding the ecommerce roles) require that `EDXAPP_ENABLE_OAUTH2_PROVIDER ` be off by default?